src/bundle/SystemInfo/Value/IbexaSystemInfo.php

```diff
@@ -49,41 +49,67 @@ class IbexaSystemInfo extends ValueObject implements SystemInfo
 
     /**
      * @var bool
+     *
+     * @uses $endOfMaintenanceDate
      */
     public $isEndOfMaintenance = true;
 
     /**
-     * @var \DateTime
+     * @var \DateTime EOM for the given release, if you have an Ibexa DXP / Enterpise susbscription.
+     *
+     * @see https://support.ibexa.co/Public/Service-Life
      */
     public $endOfMaintenanceDate;
 
     /**
      * @var bool
+     *
+     * @uses $endOfLifeDate
      */
     public $isEndOfLife = true;
 
     /**
-     * @var \DateTime
+     * @var \DateTime EOL for the given release, if you have an Ibexa DXP susbscription.
+     *
+     * @see https://support.ibexa.co/Public/Service-Life
      */
     public $endOfLifeDate;
 
+    /**
+     * @var bool If install should have subscription data (due to enterpise packages), but it's not present.
+     */
+    public $shouldHaveSubscription;
+
     /**
      * @var bool
      */
     public $isTrial = false;
 
     /**
+     * Lowest stability found in install (packages / minimumStability).
```

Comment:
Install is a verb, not a noun.

```suggestion
     * Lowest stability found in installation (packages / minimumStability).
```

Reply:
Fixed (738a175c)